pkg/framework/guest.go

```diff
@@ -264,3 +273,20 @@ func (g *Guest) WaitForNodesReady(ctx context.Context, expectedNodes int) error
 	g.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waited for %d k8s nodes to be in %#q state", expectedNodes, v1.NodeReady))
 	return nil
 }
+
+func (g *Guest) EnsureNamespaces(ctx context.Context, requiredNamespaces []string) error {
```

Comment:
I would probably name that EnsureNamespacesExists to be more clear on what this method actually does.
Also, the requirednamespaces could be renamed to namespaces imo.
WDYT?

Also, do you have a code review guide I could check?

Reply:
Thanks for the review. I'm still working on it (build fails on things I didn't work on).

Reply:
> Also, do you have a code review guide I could check?

No sorry, don't know if it exists, maybe try asking on #ask-me-anything

Reply:
Function and variable renamed